frontend/src/views/HomeView.vue

```diff
@@ -1 +1,56 @@
-<template><h1>Test</h1></template>
+<script setup lang="ts">
+import { ElButton, ElInput, ElMessage, ElFormItem, ElDialog, ElForm } from 'element-plus'
+import { reactive, ref } from 'vue'
+import axios from 'axios';
+
+const dialogFormVisible = ref(false)
+const formLabelWidth = '140px'
+const form = reactive({
+  name: '',
+  
+})
+
+const saveForm = (roleId:number) => 
+{
+  axios.put(`/api/roles/${roleId}`, form)
+    .then(response => {
+      // Trate a resposta do backend
+      console.log(response.data);
+      // Exiba uma mensagem de sucesso para o usuário
+      ElMessage({
+        message: response.data.message,
+        type: 'success'
+      });
+      dialogFormVisible.value = false;
+    })
+    .catch((error:any) => {
+      // Se ocorrer um erro, exiba uma mensagem de erro para o usuário
+      console.error('Error updating role:', error);
+      ElMessage({
+        message: 'Failed to update role',
+        type: 'error'
+      });
+    });
+};
+  dialogFormVisible.value = false;
+  const roleId = ref(0);
+</script>
+
+<template>
+  <ElButton link style="color: #29517A;" @click="dialogFormVisible = true ">
```

Comment:
Don't use inline styling, always classes. All the colors you need for this project are in colors.scss file